tests/test_disturbance.py

```diff
@@ -130,11 +129,9 @@ def test_write_disturbance_data_buffer_exception(disturbance):
         disturbance.write_disturbance_data([0] * disturbance.max_sample_number)
 
 
-@pytest.mark.skip("Exception is not implemented yet")
 @pytest.mark.soem
 @pytest.mark.eoe
 def test_write_disturbance_data_not_configured(disturbance):
-    # TODO Add exception in function for this test case
```

Comment:
The idea of this test was to check if a user tries to write disturbance data and no registers are mapped as disturbance registers.